### PR TITLE
Fix/profile joined projects

### PR DESCRIPTION
**Pull request by Columbium41**
## Description
- Fixed profile joined projects showing current logged in user's joined projects
- Refactored profile joined projects to look like repository cards instead of a table

## Screenshots
Screenshot_2023-07-05_at_3 12 10_PM

### CODE REVIEW

File: app/views/users/_repositories.html.erb [hunk 1]
```diff
@@ -1,5 +1,9 @@
-<% if @repositories.empty? or params[:show].eql?('makes') and @makes.empty? %>
+<% if params[:show].eql?('repositories') && @repositories.empty? %>
```
Comment (art29) [inline app/views/users/_repositories.html.erb:1]: why not params[:show] == 'repositories' ?

File: app/views/users/_repositories.html.erb [hunk 2]
```diff
@@ -45,23 +49,29 @@
...
-                    <td class="titlepp">
-                        <%= joined_project.project_proposal.title %>
-                    </td>
-                    <td>
-                        <%= link_to 'Show', joined_project.project_proposal, class: 'green-button' %>
-                    </td>
-                </tr>
-            </tbody>
-        </table>
+        <% project_proposal = joined_project.project_proposal %>
```
Comment (art29) [inline app/views/users/_repositories.html.erb:52]: Really not a fan of Ruby in HTML... For `has_user` you can create a method in the model

**Review by art29 — CHANGES_REQUESTED** [review]
Few things to fix

File: app/views/users/_repositories.html.erb (hunk 1 above)
Comment (Columbium41) [reply app/views/users/_repositories.html.erb:1]: I believe that was how it was done, so I continued to do it this way. I can change it to use double equals

**Review by art29 — APPROVED** [review]

**Pull request merged**